libraries/Microsoft.Bot.Builder/Adapters/TestFlow.cs

```diff
@@ -165,6 +165,36 @@ public TestFlow AssertReply(string expected, string description = null, uint tim
             return AssertReply(_adapter.MakeActivity(expected), description, timeout);
         }
 
+        /// <summary>
+        /// Adds an assertion that the turn processing logic responds as expected.
+        /// </summary>
+        /// <param name="expected">The part of the expected text of a message from the bot.</param>
+        /// <param name="description">A message to send if the actual response is not as expected.</param>
+        /// <param name="timeout">The amount of time in milliseconds within which a response is expected.</param>
+        /// <returns>A new <see cref="TestFlow"/> object that appends this assertion to the modeled exchange.</returns>
+        /// <remarks>This method does not modify the original <see cref="TestFlow"/> object.</remarks>
+        /// <exception cref="Exception">The bot did not respond as expected.</exception>
+        public TestFlow AssertReplyContain(string expected, string description = null, uint timeout = 3000)
```

Comment:
Please add unit tests for this addition, we don't have a TestFlowTests class at the moment (and we would like to start adding more coverage) so go ahead and create one in the Microsoft.Bot.Builder.Tests project under the Adapters folder (needs to be created).

Reply:
Added tests in https://github.com/microsoft/botbuilder-dotnet/pull/1922/commits/0a3d14ccc374d3f6c7864ef8a0a55e69381b8c3b